### ISSUE TITLE
Inquiry: Phantasy Star Collection emulation challenges

### ISSUE THREAD

**Issue by chipzoller** [mgba-emu/mgba#1883]
More of a question than an issue, but Phantasy Star Collection for GBA seems to be quite problematic across the board from an emulation perspective, most notably the first in the series. It suffers from audio issues, flickering, and others, and not just in mGBA but other emulators as well. Since I didn't see any older issues that talked about this, I'm just curious what the difficulties are if it has ever been investigated.

**Comment by endrift** [comment]
The flickering is not a bug. It is a trick that many games do to take advantage of hardware screen ghosting to antialias. In this case, it's doing it because the "emulated" game has a higher resolution vertically than the GBA, so it does it on scanlines that it deletes to have them "partially" appear. This can be mitigated by using interframe blending.

The only audio issue I'm aware of is the Digital Eclipse logo, which occurs because the game uses BIOS calls for audio that are not implemented in the built-in BIOS. Using an official BIOS dump should solve that.

Are there any I'm missing?

**Comment by chipzoller** [comment]
The other and more prominent issue I've noticed is in the first game, right at the beginning when the audio pauses in silence for a couple seconds, the entire game seems to slow significantly. Observed this on multiple version of mGBA and the US edition of the game (separate dumps with the same hash).

**Comment by endrift** [comment]
I'll take a look.

**Comment by chipzoller** [comment]
I went back and tested mGBA 0.8.3 on Win64 as well as VisualBoyAdvance-M 2.1.4 and neither have the lag issue, but mGBA has audio "anomalies" (do not know how to technically refer to the issue) in that same moment of silence where there is lag on the 3DS version.

**Comment by RetroEdit** [comment]
Do those audio anomalies reproduce in the latest dev build (https://mgba.io/builds/1/)? There have been some audio changes since 0.8.3.

**Comment by chipzoller** [comment]
Yes, still the same anomalies, and in addition there are graphical issues with the intro in the first game where none existed prior.
![image](https://user-images.githubusercontent.com/29008123/92933985-ef53ef00-f414-11ea-9fe8-69641707b2a5.png)

**Comment by chipzoller** [comment]
Tested with 0.9.3 and visual issues seem to be smoothed out but the audio issues described above still seem to be present on 3DS. Great work you've done with this project, so thank you for all your accomplishments!
